Scripts/git-clone-pull.sh

```diff
@@ -4,14 +4,20 @@
 # Description: Shell script to clone a git repository and pull the latest changes
 # Authors: Robbie, and Swamy
 # Date: 12-Mar-2022
-# Modified: 14-Mar-2022
+# Modified: 15-Mar-2022
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # 
 
+if [[ "/$PWD/" != *"/eshop-main/Scripts"* ]]; then
```

Comment:
if we are warning people to execute the script from Scripts, the trailing star is not needed